Extensions/CodeGeneration/CodeGenerationDisplayDriver.cs

```diff
@@ -134,14 +134,14 @@ private string ConvertJToken(JToken jToken)
 
                 case JArray jArray:
                     var token = string.Join(", ", jArray.Select(ConvertJToken));
-                    var format = token.Contains("ListValueOption") ? "\n\t\t\t" : "";
+                    var format = token.Contains("ListValueOption") ? "\n    " : "";
 
                     return $"new[] {format}{{ {string.Join(", ", token)} {format}}}";
 
                 case JObject jObject:
                     if (jObject["name"] != null && jObject["value"] != null)
                     {
-                        return $"\n\t\t\t\tnew ListValueOption {{ Name = \"{jObject["name"]}\", Value = \"{jObject["value"]}\" }}";
+                        return $"\n               new ListValueOption {{ Name = \"{jObject["name"]}\", Value = \"{jObject["value"]}\" }}";

```

Comment:
The indentation here and above might not be static as it depends on how deep we are, see `indentation` in `AddSettingsWithout()`. I think it should pass this value to `ConvertJToken()` and we should use it as the base indentation here.

Reply:
Finally coming back to this. We will need to maintain the recursive depth of calls to `ConvertJToken()` in order to determine the appropriate depth? Is that correct?

Reply:
Yes, see how `AddSettingsWithout()` does it.